java_cursor.txt

```diff
@@ -0,0 +1 @@
+Y3Vyc29yOjEwMA==
```

Comment:
please move cursors to `data/repo_meta` or create a new subdir `data/repo_cursors`. please document the usage of the cursors somewhere in `README.md`